meta-mender-core/conf/systemd-boot/README.md

```diff
@@ -0,0 +1,111 @@
+# systemd-boot reference configuration
+
+This configuration demonstrates support for systemd-boot on a QEMU system
+via the OVMF UEFI firmware.
+
+As tested, this systemd-boot support requires the meta-intel layer
+to create suitable UEFI images.
+
+## Setup
+
+The `dunfell.xml` and `master.xml` files may be used with the `repo` tool
+to clone all of the required layers and projects required.
+
+In your top-level BSP directory:
+
+```shell
+branch="dunfell"
+repo init -u "https://github.com/zach-welch-timesys/meta-mender" \
```

Comment:
Yes, but this commit/content needs to be moved elsewhere, yes?  It was included here to allow testing, which prior to merging would require my staging repository. After merging, this content gets located somewhere more permanent and it can be changed to its proper location. Thanks for calling it out here.

Reply:
Yes, you're right, I had forgotten about this detail. For my own reference, I'm re-posting the relevant discussion from #1474:

>> In `meta-mender-core/conf/systemd-boot/`, an example configuration has been provided to verify the changes using QEMU (and, with a few tweaks, on real hardware). This configuration has been used to demonstrate that the mender-client integration works. Instructions for building and testing this configuration can be found in the README therein, along with some notes about the current state of the branch changes. (This patch likely should be dropped, with the content moved to more suitable locations.)
>
> Agreed. Since this is, at least for the moment, community maintained, this might fit better on [Mender Hub under Board Integrations](https://hub.mender.io/c/board-integrations/yocto-project/10). I've added a few code comments about these files below, but not all are relevant if they are moved.

If you have no objections I propose putting this README on Mender Hub, and we can link to it from here.